include/deal.II/grid/tria.h

```diff
@@ -2547,6 +2550,35 @@ class Triangulation : public Subscriptor
    * @}
    */
 
+  /*---------------------------------------*/
+  /*---------------------------------------*/
+
+  /**
+   * @name Vertex iterator functions
+   * @{
+   */
+
+  /**
+   * Iterator to the first used vertex.
+   */
+  vertex_iterator        begin_vertex() const;
+
+  /**
+   * Iterator to the first active vertex. Because all vertices are active,
```

Comment:
Yes, that's wrong. Not all vertices are active. If you think about starting with one cell, refining it, then coarsening it again, you end up with a bunch of slots in the `vertices` array that still exist but are unused.